Heroes.SDK.Library/Definitions/Enums/Custom/Players.cs

```diff
@@ -7,4 +7,178 @@ public enum Players : int
         Three,
         Four
     }
+
+   public enum PlayerMode
+    {
```

Comment:
Needs merging with https://github.com/Heroes-Hacking-Central/Heroes.SDK/blob/6933109997480ff84c7f4af94f63de38f86c71e5/Heroes.SDK.Library/Definitions/Structures/Player/PlayerTopComponents/Action.cs#L3

Reply:
Done in 67e9b8ea528cd6fc631a49a9b02f4e0fc4170388